### PR TITLE
Repository cleanup

### PR DESCRIPTION
**Pull request by vincenzopalazzo**
- [X] Removing travis config file
- [X] Update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,34 @@
-# Rust c-lightning client
+<div align="center">
+  <h1>Rust c-lightning client</h1>
 
-[![Status](https://travis-ci.org/laanwj/rust-clightning-rpc.png?branch=master)](https://travis-ci.org/laanwj/rust-clightning-rpc)
-[![Crates.io](https://img.shields.io/crates/v/clightningrpc.svg)](https://crates.io/crates/clightningrpc)
-[![Docs.rs](https://docs.rs/clightningrpc/badge.svg)](https://docs.rs/clightningrpc/)
+  <p>
```
Comment (laanwj) [inline README.md:4]: Any reason to use html here instead of markdown ?

Comment (vincenzopalazzo) [reply README.md:4]: The only reason is that with HTML I can center the text and make the style of the repo introduction different from the style of it. If you prefer to have the introduction of the readme with the same format, I can reintroduce the markdown again

Comment (laanwj) [reply README.md:4]: Okay, seems ok to keep it like this to me then.

File: README.md [hunk 2]
```diff
@@ -65,6 +91,13 @@ compile, sorry!
 - Call `rustfmt src/lib.rs examples/*.rs` before committing
 - If you can, GPG-sign at least your top commit when filing a PR
 
+# Supports
+
+If you want support this library considered to donate with the following methods
```
[the remark this answers is not in the document]
Comment (vincenzopalazzo) [reply README.md]: ops, thanks for this fix, now should be ok I think

**Pull request merged**